lib/run-in-terminal.coffee

```diff
@@ -180,37 +180,55 @@ module.exports =
                 selectors.tree,
                 "tree-start-terminal-here",
                 "Start terminal here [tree]",
-                () => @tree_start_terminal_here(false)
+                () => @tree_start_terminal_here(false, false)
             ],
             [
                 selectors.tree,
                 "tree-start-terminal-here-and-run",
                 "Start terminal here and run [tree]",
-                () => @tree_start_terminal_here(true)
+                () => @tree_start_terminal_here(true, false)
+            ],
+            [
+                selectors.tree,
+                "tree-start-terminal-here-and-run-with-par",
+                "Start terminal here and run [tree] (with par)",
```

Comment:
![default](https://cloud.githubusercontent.com/assets/9821835/12744143/99f30c58-c9a3-11e5-8717-a81641366570.png)

I think you use very long names. Maybe you should change it to `Start terminal`, `Run in terminal` and `Run in terminal with parameters`. Maybe this name will have some sorting in menu problem? But `[tree]`, `[editor]` and etc - this is trash )))